### ISSUE TITLE
Vector of Wiener processes

### ISSUE THREAD

**Issue by julianiacoponi** [patrick-kidger/diffrax#355]
I want to try and use `diffrax` to simulate some correlated ("directional") noise in a 3D oscillator i.e. solving the following SDEs

![image](https://github.com/patrick-kidger/diffrax/assets/32678459/d458afe9-048a-49c4-b35f-4a11a2d6857e)

I couldn’t parse from the docs what kind of solver would treat the noise term as a matrix and the dW term as a vector of Wiener processes, at the same time.

Does diffrax have such a solver / can I build one from what `diffrax` offers?

**Comment by hsimonfroy** [comment]
Hi,
You can use
```python
diffusion

**Comment by patrick-kidger** [comment]
Yup! `diffrax.ControlTerm` is the way to implement a matrix-valued diffusion with vector-valued noise.

If this is unclear then I'd definitely be happy to consider a PR improving the documentation. :)
